src/main/java/duke/task/TaskList.java

```diff
@@ -119,4 +119,15 @@ public void deleteTask(String[] replyArr) throws DukeException {
         }
     }
 
+    public void findTaskByKeyword(String[] replyArr) {
```

Comment:
Maybe you'd like to implement `A-Varargs` here? Instead of `String[] replyArr`, you can use `String... replyArr` and some minor tweaks to the logic below!